packages/components/src/components/marketplace-list.tsx

```diff
@@ -103,50 +62,65 @@ export const MarketplaceList = ({
   }, [items, pageSize]);
 
   return (
-    <section className={clsx('w-full bg-white dark:bg-dark', className)}>
+    <section
+      className={cn(
+        'MarketplaceSearch',
+        colorScheme,
+        'w-full dark:bg-neutral-900 [&.green]:bg-green-1000',
+        className,
+      )}
+    >
       {title && (
-        <h2 className="mb-4 mt-0 text-xl font-bold text-black md:text-2xl dark:text-gray-50">
+        <Heading
+          as="h2"
+          size="sm"
+          className="mb-6 mt-4 text-2xl/8 font-medium text-[--fg,theme(colors.neutral.900)] dark:text-white"
+        >
           {title}
-        </h2>
+        </Heading>
       )}
       {pages[currentPage]?.length ? (
         <>
-          <table className="w-full border-collapse">
-            <thead className="whitespace-nowrap px-2 text-left text-xs font-semibold uppercase text-gray-300 dark:text-gray-600">
-              <tr className="border-0">
-                <th className="px-2" />
-                <th className="px-2">Name</th>
-                <th className="hidden px-2 md:table-cell">Last Update</th>
-                <th className="px-2" />
-              </tr>
-            </thead>
-            <TableBody items={pages[currentPage]} />
-          </table>
+          <ul ref={listRef} className="grid gap-4 lg:grid-cols-2 lg:gap-6">
+            {pages[currentPage].map((item, i) => {
+              return (
+                <li key={item.title} className="*:h-full">
+                  <MarketplaceListItem
+                    item={item}
+                    tabIndex={i === 0 ? 0 : -1}
+                    // focusgroup vertical navigation
+                    onKeyDown={event => {
+                      const ul = listRef.current;
```

Comment:
```suggestion
                      const ul = event.target.parentElement!.parentElement;
```
or `currentTarget` (if you have type errors)
to avoid adding `ref`?

Reply:
cc @hasparus  `ul` always exists

Reply:
changing to parentElement right now